app/src/pages/HomePage.spec.ts

```diff
@@ -1,89 +1,145 @@
+} from "@/tests/mockdata";
+import waitForExpect from "wait-for-expect";
+import { appLanguageIdAsRef, initLanguage } from "@/globalConfig";
 
 vi.mock("@auth0/auth0-vue");
-vi.mock("luminary-shared", async () => {
-    const actual = await vi.importActual("luminary-shared");
-    return {
-        ...actual,
-        db: {
-            someByTypeAsRef: vi.fn(),
-            tagsWhereTagTypeAsRef: vi.fn(),
-        },
-    };
-});
+vi.mock("vue-router");
 
-// Mock the IgnorePagePadding and HorizontalScrollableTagViewer components
-vi.mock("@/components/IgnorePagePadding.vue", () => ({
-    default: { template: "<div><slot /></div>" },
-}));
-vi.mock("@/components/tags/HorizontalScrollableTagViewer.vue", () => ({
-    default: {
-        props: ["title", "queryOptions", "tag"],
-        template: '<div>{{ title }} {{ tag ? tag.name : "" }}</div>',
-    },
-}));
-
-describe("HomePage", () => {
-    beforeEach(() => {
-        setActivePinia(createPinia());
-        (auth0 as any).useAuth0 = vi.fn().mockReturnValue({
-            isLoading: ref(false),
-            isAuthenticated: ref(true),
-        });
-        (db.someByTypeAsRef as any).mockReturnValue(true);
-        (db.tagsWhereTagTypeAsRef as any).mockImplementation(
-            (tagType: TagType.Category, options: queryOptions) => {
-                if (options.filterOptions?.pinned) {
-                    return [{ _id: "1", name: "Pinned Category" }];
-                }
-                return [
-                    { _id: "2", name: "Unpinned Category" },
-
-                    // empty category
-                    { _id: "2", name: "" },
-                ];
-            },
-        );
+describe("HomePage.vue", () => {
+    beforeAll(() => {
+        accessMap.value = viewAccessToAllContentMap;
+        initLanguage();
     });
 
-    afterEach(() => {
-        vi.clearAllMocks();
+    beforeEach(async () => {
+        await db.docs.bulkPut([mockLanguageDtoEng, mockLanguageDtoFra, mockLanguageDtoSwa]);
     });
 
-    it("displays a message when there are no contents", async () => {
-        (db.someByTypeAsRef as any).mockReturnValue(false);
+    afterEach(async () => {
+        vitest.clearAllMocks();
+        await db.docs.clear();
+        await db.localChanges.clear();
+    });
 
-        const wrapper = mount(HomePage);
+    describe("No content notifications", () => {
+        it("renders correctly with no content and not authenticated", () => {
+            (auth0 as any).useAuth0 = vi.fn().mockReturnValue({
+                isAuthenticated: ref(false),
+            });
+            const wrapper = mount(HomePage);
 
-        expect(wrapper.text()).toContain(
-            "You don't have access to any content. If you believe this is an error, send your contact person a message.",
-        );
-    });
+            expect(wrapper.text()).toContain(
+                "There is currently no content available. If you have an account, first  log in.",
```

Comment:
Change "If you have an account, first log in"
to
"Please log in if you have an account."

Sorry, made comment in test file. But please update in Vue component as well?

Reply:
Ok